plugin/plugin_defs/defs.go

```diff
@@ -0,0 +1,55 @@
+package plugin_defs
```

Comment:
let’s keep this in `plugin.go`, including `pipeline.go` and `util.go` under `plugin/` dir, and only move the tool-specific logic to a separate folder

Reply:
rest looks good